### PR TITLE
http function to verify signed message

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: apps/functions/src/app/player.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { utils } from 'ethers';
+
+export const checkSignature = async (data: { message: string, signature: string, tokenId: string }): Promise<void> => {
```
Comment (GrandSchtroumpf) [inline apps/functions/src/app/player.ts]: In a next PR create an interface that we could reuse in the frontend too: 
- Create a lib `model`
- Add an alias `@model` in tsconfig
- Create the interface `SignedMessage`
- Use it in functions & player

File: apps/marketplace/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -30,6 +30,10 @@ const routes: Routes = [
   {
     path: 'faq',
     loadChildren: () => import('./faq/faq.module').then(m => m.FAQModule)
+  },
+  {
+    path: 'player',
```
Comment (GrandSchtroumpf) [inline apps/marketplace/src/app/app-routing.module.ts]: This should be in the player app not the marketplace.
Also add the tokenId as params.

File: apps/functions/src/app/player.ts (hunk 1 above)
Comment (phpgeekfr) [reply apps/functions/src/app/player.ts]: ok will do !

File: apps/player/src/app/watch/watch.component.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { MetamaskService } from '@nft/metamask';
+import { AngularFireFunctions } from '@angular/fire/functions';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+  selector: 'nft-watch',
+  templateUrl: './watch.component.html',
+  styleUrls: ['./watch.component.scss']
+})
+export class WatchComponent implements OnInit {
```
Comment (GrandSchtroumpf) [inline apps/player/src/app/watch/watch.component.ts]: Pas besoin de mettre le tokenId en property
```typescript
getPlayerId = this.functions.httpCallable<SignedMessage, string>('getPlayerId');

async signMessage() {
 try {
   const tokenId = this.route.snapshot.paramMap.get('tokenId') as string;
   const message = '...';
   const signature = await this.metamaskService.signMessage(message);
   this.playerUrl = await this.getPlayerId({ tokenId, message, signature }).toPromise();
 } catch (err) { ... }
}
```

**Review by GrandSchtroumpf — APPROVED** [review]

File: apps/functions/src/app/player.ts (hunk 1 above)
Comment (phpgeekfr) [reply apps/functions/src/app/player.ts]: associated  PR https://github.com/blockframes/nft/pull/14

**Pull request merged**